builtin/providers/aws/resource_aws_elb.go

```diff
@@ -271,6 +283,9 @@ func resourceAwsElbRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("listener", flattenListeners(lb.ListenerDescriptions))
 	d.Set("security_groups", lb.SecurityGroups)
 	d.Set("subnets", lb.Subnets)
+	d.Set("idle_timeout", lbAttrs.ConnectionSettings.IdleTimeout)
+	d.Set("connection_draining", lbAttrs.ConnectionDraining.Enabled)
+	d.Set("connection_draining_timeout", lbAttrs.ConnectionDraining.Timeout)
```

Comment:
`Timeout` is apparently [not a required field](http://docs.aws.amazon.com/ElasticLoadBalancing/latest/APIReference/API_LoadBalancerAttributes.html), at least according to the documentation. If it's not set, do we get a value here? Just curious. `d.Set` _should_ prevent a nil pointer dereference